stdlib/xml/dom/expatbuilder.pyi

```diff
@@ -1,3 +1,105 @@
-from typing import Any
+# NOTE: Incomplete, lots of unannotated parameters currently
 
-def __getattr__(name: str) -> Any: ...  # incomplete
+from typing import Any, NoReturn
+from typing_extensions import Literal
+from xml.dom.minidom import Document, DOMImplementation, Node, TypeInfo
+from xml.dom.xmlbuilder import DOMBuilderFilter, Options
+
+TEXT_NODE = Node.TEXT_NODE
+CDATA_SECTION_NODE = Node.CDATA_SECTION_NODE
+DOCUMENT_NODE = Node.DOCUMENT_NODE
+FILTER_ACCEPT = DOMBuilderFilter.FILTER_ACCEPT
+FILTER_REJECT = DOMBuilderFilter.FILTER_REJECT
+FILTER_SKIP = DOMBuilderFilter.FILTER_SKIP
+FILTER_INTERRUPT = DOMBuilderFilter.FILTER_INTERRUPT
+theDOMImplementation: DOMImplementation | None
+
+class ElementInfo:
+    tagName: Any
+    def __init__(self, tagName, model: Any | None = ...) -> None: ...
+    def getAttributeType(self, aname) -> TypeInfo: ...
+    def getAttributeTypeNS(self, namespaceURI, localName) -> TypeInfo: ...
+    def isElementContent(self) -> bool: ...
+    def isEmpty(self) -> bool: ...
+    def isId(self, aname) -> bool: ...
+    def isIdNS(self, euri, ename, auri, aname) -> bool: ...
+
+class ExpatBuilder:
+    doc: Document
+    curnode: Document
```

Comment:
- It took me a while to realize how these work. Maybe add a comment, e.g. `# created in reset()`?
- `curnode` sounds like it can be any node whatsoever, not necessarily a node that represents the entire document.
- Typo: it should be `curNode`, not `curnode`

Reply:
Do you reckon it should be `curNode: Node`, or `curNode: Any`?

Reply:
I'm fine with both.

Reply:
Done!